greedybear/consts.py

```diff
@@ -7,6 +7,7 @@
 POST = "POST"
 
 FEEDS_LICENSE = "https://github.com/honeynet/GreedyBear/blob/main/FEEDS_LICENSE.md"
+SKIP_FEED_VALIDATION = False
```

Comment:
- if you want to dynamically change this without having to re-build the code, you can add it as an environment variable in `env_file_template` and then extract it as a Django constant in `settings.py`

Reply:
Great idea, thanks! :) Will do that and merge.